templates/frontpage_guest.mustache

```diff
@@ -64,12 +64,8 @@
             {{^sliderenabled}}
                 <div class="row" id="loginbox">
                     <div class="col-sm-12 col-md-7 hidden-sm-down" id="intro">
-                        {{#bannerheading}}
-                            <h1>{{{ bannerheading }}}</h1>
-                        {{/bannerheading}}
-                        {{#bannercontent}}
-                            <h2>{{{ bannercontent }}}</h2>
-                        {{/bannercontent}}
+                        <h1>Perfect Learning System</h1>
```

Comment:
This value should be set in the language strings file. Similar to the forgotten password field.

Reply:
It comes from `settings.php`

Please use those strings instead of hard-coding them:
```
    // Bannerheading.
    $name = 'theme_moove/bannerheading';
    $title = get_string('bannerheading', 'theme_moove');
    $description = get_string('bannerheadingdesc', 'theme_moove');
    $default = 'Perfect Learning System';
    $setting = new admin_setting_configtext($name, $title, $description, $default);
    $setting->set_updatedcallback('theme_reset_all_caches');
    $page->add($setting);
```

Reply:
I think it comes from the themes language strings file, settings.php uses those values.

https://github.com/SecurityCompass/moodle-theme_moove/blob/fa232fe27294ece1bb9ec3776351c44a9644aec5/lang/en/theme_moove.php#L91

Reply:
Though I have changed in settings.php, bannerheading and bannercontent are fetched from database. Below are the sql queries. 

```
update moodle.mdl_config_plugins set value="Perfect Learning System" where plugin="theme_moove" AND name="bannerheading" limit 1;
update moodle.mdl_config_plugins set value="Our training suites in software security cover OWASP Top 10, secure software design, secure software coding, and other courses that cater to specific roles in order to help improve an organization’s security posture. Whether your team is responsible for developing mobile or web applications securely, at Security Compass, we've got you covered." where plugin="theme_moove" AND name="bannercontent" limit 1;
```